senza/templates/_helper.py

```diff
@@ -180,20 +212,36 @@ def check_iam_role(application_id: str, bucket_name: str, region: str):
                     AssumeRolePolicyDocument=json.dumps(assume_role_policy_document),
                 )
 
-    update_policy = bucket_name is not None and (
-        not exists
-        or confirm(
-            "IAM role {} already exists. ".format(role_name)
-            + "Do you want Senza to overwrite the role policy?"
+    attach_mint_read_policy = bucket_name is not None and (
+        (not exists and create)
+        or (
+            exists and click.confirm(
+                "IAM role {} already exists. ".format(role_name)
+                + "Do you want Senza to overwrite the role policy?"
+            )
         )
     )
-    if update_policy:
+    if attach_mint_read_policy:
         with Action("Updating IAM role policy of {}..".format(role_name)):
-            policy = get_iam_role_policy(application_id, bucket_name, region)
+            mint_read_policy = create_mint_read_policy_document(application_id, bucket_name, region)
             iam.put_role_policy(
                 RoleName=role_name,
                 PolicyName=role_name,
-                PolicyDocument=json.dumps(policy),
+                PolicyDocument=json.dumps(mint_read_policy),
+            )
+
+    cross_stack_policy_exists = False
+    cross_stack_policy_name = "CrossStackPolicy-{}".format(application_id)
```

Comment:
Let's use the name `system-cf-notifications` for the policy to match other existing tools

Reply:
Changed